### DISCUSSION TITLE
Custom queryset on Admin change list

### DISCUSSION THREAD

**Discussion by girase1pranay** [jrief/django-admin-sortable2#401] [Q&A]
In my application I have added support for "Soft Delete" for object, let say I have Y which is object of X Model, if I delete Y, it doesn't get deleted from database instead it's status get changes to 'D'(Deleted), and in listing screen I have overridden the get_queryest method which will exclude objects with status as 'D' (Deleted).
In this case whatever ordering is set to Y object, it will stay as it is and affect the ordering in case of order changing with actions "move selected ... pages back" and others... 
Reordering doesn't work as per expectations in above scenario. 
How can I overcome from this issue?

**Comment by jrief** [comment]
This is where "grouping" as generic solution comes into play. It is a feature I always wanted to implement.
